### PR TITLE
Update private-channels-life-cycle-management.md

### PR DESCRIPTION
**Pull request by dariomws**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/3733

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/private-channels-life-cycle-management.md [hunk 1]
```diff
@@ -180,38 +180,16 @@ As an admin, you can use PowerShell or Graph APIs commands to query these URLs.
...
-2. Run the following, where &lt;group_id&gt; is the group Id of the team and &lt;channel_id&gt; is the channel Id.
-
-    **Request**
-
-    ```PowerShell
-    Get-TeamChannelUser -GroupId <group_id> -MembershipType Private -DisplayName "<channel_name>" 
-    ```
-    
-    **Response**
+2. Run the following, where &lt;group_id&gt; is the group Id of the team and &lt;channel_name&gt; is the channel name.
```
Comment (JohanFreelancer9) [inline Teams/private-channels-life-cycle-management.md]:
```suggestion
2. Run the following, where &lt;group_id&gt; is the Group ID of the team and &lt;channel_name&gt; is the channel name.
```

File: Teams/private-channels-life-cycle-management.md [hunk 2]
```diff
@@ -180,45 +180,23 @@ As an admin, you can use PowerShell or Graph APIs commands to query these URLs.
...
-2. Run the following, where &lt;group_id&gt; is the group Id of the team and &lt;channel_id&gt; is the channel Id.
-
-    **Request**
-
-    ```PowerShell
-    Get-TeamChannelUser -GroupId <group_id> -MembershipType Private -DisplayName "<channel_name>" 
-    ```
-    
-    **Response**
+2. Run the following, where &lt;group_id&gt; is the Group ID of the team and &lt;channel_name&gt; is the channel name.
```
Comment (JohanFreelancer9) [inline Teams/private-channels-life-cycle-management.md]:
```suggestion
2. Run the following, where &lt;group_id&gt; is the group ID of the team and &lt;channel_name&gt; is the channel name.
```

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by LolaJacobsen** [comment]
Checking with PM Roshin Ramesan.

**Comment by LolaJacobsen** [comment]
Checking with PM Roshin Ramesan.

**Comment by LolaJacobsen** [comment]
Roshin referred me to Arun Das. I've ping'd him - both on the 14th and today.

**Comment by scanum** [comment]
@LolaJacobsen Thanks Lola

**Comment by LanaChin** [comment]
I had also pinged the PMs on this 2/10.

**Comment by LanaChin** [comment]
@dariomws - The -MembershipType Private parameter is in Teams PowerShell module from the PowerShell Test Gallery. The article includes steps for how to install the module from the Test Gallery except the link in Step 1 of this section, https://docs.microsoft.com/microsoftteams/private-channels-life-cycle-management#using-powershell-2, is incorrect. Instead of https://www.powershellgallery.com/packages/MicrosoftTeams, it needs to be changed to https://www.poshtestgallery.com/packages/MicrosoftTeams.

Can you revert the changes and update the URL? Thank you!

**Comment by dariomws** [comment]
@LanaChin oh it's true! My mistake.

I have reverted the -MembershipType and updated the URL. Thanks!

**Comment by dariomws** [comment]
@LanaChin Sorry Lana, I double checked it and couldn't find -MembershipType parameter on module version 1.0.20.

**Comment by LanaChin** [comment]
thanks @dariomws - thought about this further and to avoid any confusion, let's just remove step 1 from that section. The article already includes a note at the top about using the pre-release module from the Test Gallery and re-iterates it in the section for how to install the module.  Also, the other PowerShell sections don't include that step. I've made that update and merged.

**Comment by LanaChin** [comment]
@dariomws, you're correct. Thanks for work on this. Confirmed that the -MembershipType Private parameter isn't needed for Get-TeamChannelUser and Add-TeamChannelUser. The doc has been updated: https://docs.microsoft.com/en-us/MicrosoftTeams/private-channels-life-cycle-management#list-and-update-roles-of-owners-and-members-in-a-private-channel.

**Pull request merged**